src/components/navbars/NavbarSide.vue

```diff
@@ -0,0 +1,51 @@
+<script lang="ts" setup>
+import { Icon } from '@iconify/vue';
+import { useI18n } from 'vue-i18n';
+
+import messages from "../../i18n/index"
+
+const i18n = useI18n({
+  messages,
+  useScope: "local"
+})  
+
+const {t} = i18n;
+
+interface NavLink {
+  tKey: string,
+  to: string,
+  icon: string
+}
+
+const links: NavLink[] = [
+  {tKey: "navbar.tricks", to: "/tricks", icon: "ic:baseline-auto-awesome"},
+  {tKey: "navbar.combos", to:"/combos", icon: "ic:baseline-spoke"},
+  {tKey: "navbar.glossary", to: "/glossary", icon: "ic:sharp-menu-book"},
+  {tKey: "navbar.generator", to: "/generator", icon: "ic:sharp-factory"},
```

Comment:
`/combos/new` or `/combos/new/generate` would be more descriptive and would more closely model REST-Like Structures

Reply:
Agreed. I think `/combos/new` would not be good, as to me that route seems more appropriate for the page where a user can create (not generate) a custom combo. 
Would `/combos/new/generate` be better in any way than `/combos/generate`? Otherwise I'd go with the `/combos/generate` option.

Reply:
The question is… in what way to we wish to implement generation.

If it is logically a subpage of creating a new combo, e.g. you click on "Create new" and then click a Dice-Button or something like that, I would vouch for `/combos/new/generate`. 

If `/combos/new` and `/combos/generate` can be seen as fundamentally different routes, `/combos/generate` would work. 

To me it seems that generation would be fundamentally tied to creation. But it really depends on the impl.


I was thinking about a Settings-Button on `/combos/new` which would create a popup where you can select your params and a Dice-Button which would reroll for a new Combo.

The "result" would be placed into the List of Tricks which you could also get by manually adding tricks to the combo. 
In that sense, generation would basically be pre-filling the contents on the `/combos/new` page and you wouldnt even need a `/combos/new/generate` 


So yeah… depends on the impl :P

Reply:
From all that I have seen and heard so far, the Combo Generator will be a very separate feature with its own tab in the navbar. The process currently (and most probably going forwards) is, that you generate a combo and in case you like it a lot you can save it, which is done by then opening the menu for manually creating a combo and automatically filling in the tricks from the generator.

Therefore, if I read your answer correctly, using `/combos/generate` should be alright shouldn't it?

Reply:
Yup. combos/generate works for me for now. 
I would like some input from the rest of the team about how they envision `generate` to work, but that's out of scope for now :) 

feel free to resolve